03_TS_events_promises/task_events/calculator.ts

```diff
@@ -0,0 +1,33 @@
+export class Calculator {
+
+    constructor(){}
+
+    public sum (first: number, second: number) : number {
+        return +first + +second;
```

Comment:
You already protect these arguments from being not a number in your `events.ts` code.